### ISSUE TITLE
Label prediction

### ISSUE THREAD

**Issue by LaurensCorbiere** [PatRyg99/eigenvector-grouping#5]
I have a trained a PointNet++ with eigenvector grouping stategy on a dataset containing point clouds with points belonging to pulmonary arteries and background. When applying the trained model to the test set, all points of each patient are labelled as artery. Furthermore, all points of all patients get exactly the same prediction for being an artery or background. So apparently, the model has learnt nothing about distinguishing points belonging to arteries from background.

Do you have some suggestions how this could have happened?

**Comment by PatRyg99** [comment]
I don't know your exact setting so I can only give some general suggestions:
* Check if other grouping strategies (mainly `MSG`) behave the same to detect whether an issue lies in the data preparation or the model itself
* Is the performance on the training set poor as well? Are you tracking performance on the validation set during training, is it poor as well?
* Check if input point clouds are normalized to the `[-0.5, 0.5]` interval as stated in the paper since grouping radii are adapted to this interval. When using `PointNet++` the grouping radii are provided in the same metric space as the input point cloud, they should be accordingly scaled when the input point cloud is of a different size.
* If the point cloud is appropriately scaled maybe you need to tweak some `EVG` parameters e.g. vector length or radius since they were adapted for the coronary artery case.
